app/src/main/java/com/example/arc_exapmle/activity/MainActivity.kt

```diff
@@ -156,28 +160,44 @@ class MainActivity : AppCompatActivity() {
 
         })
 
-        mainActivityViewModel.toastMutableLiveData.observe(this, Observer {
 
-            Toast.makeText(this, it, Toast.LENGTH_SHORT).show()
+    }
 
+    override fun onStop() {
+        super.onStop()
 
-        })
+        mainActivityViewModel.toastMutableLiveData.removeObservers(this)
```

Comment:
The live data will do it why do we need to do it again?

Reply:
yeah I know. that line is for a test that I was doing a few weeks back . I guess I forgot to delete it.